core/dbt/contracts/graph/nodes.py

```diff
@@ -1068,6 +1068,7 @@ class UnitTestNode(CompiledNode):
     attached_node: Optional[str] = None
     overrides: Optional[UnitTestOverrides] = None
     config: UnitTestNodeConfig = field(default_factory=UnitTestNodeConfig)
+    this: Optional[ModelNode] = None
```

Comment:
I'm wondering if using "this" in this way might not be confusing. I think "this" in the rest of the context is actually a Relation, right?  What do you think of calling it "original_input_node"?

Reply:
`attached_node` above (should maybe be renamed to `tested_node_unique_id`(I can go ahead and rename that here) already represents that but this (no pun intended) is actually different.

Reply:
I'm going to test something out to see if they can be used interchangeably though..

Reply:
I've updated this to explicitly include `this_input_node_unique_id` and updated `attached_node` to be `tested_node_unique_id`: https://github.com/dbt-labs/dbt-core/pull/8891/files#diff-42cfc99b2fcd89ab75957b8999bee2b1af822f7d39953ca99f1f285c170557c1R1068-R1069

the provider.this implementation needs to do a lookup now but the benefit is that we don't store the whole node on the UnitTestNode which may want to make its way to a serialized artifact